src/screens/AddSiteFormScreen.js

```diff
@@ -75,11 +79,9 @@ class AddSiteFormScreen extends Component {
 
         if (Platform.OS === 'ios') {
             return (
-                <Button
+                <NavbarButton
                     title={reset}
                     onPress={onClickReset}
-                    backgroundColor="rgba(0,0,0,0)"
-                    color={linkColorBlue}
```

Comment:
Use of new `NavbarButton` which cuts the duplication of code everywhere. Another benefit of this will be that when we get to both platforms, we can do the android/ios logic in the `NavbarButton` itself instead of in each screen.